src/common/syncjournaldb.h

```diff
@@ -47,7 +47,7 @@ class OCSYNC_EXPORT SyncJournalDb : public QObject
     Q_OBJECT
 public:
     explicit SyncJournalDb(const QString &dbFilePath, QObject *parent = nullptr);
-    virtual ~SyncJournalDb();
+    ~SyncJournalDb() override;
```

Comment:
Is that really needed? I think the destructor gets overridden by default https://en.cppreference.com/w/cpp/language/virtual